hudi-client/src/main/java/org/apache/hudi/table/action/rollback/RollbackHelper.java

```diff
@@ -71,8 +71,9 @@ public RollbackHelper(HoodieTableMetaClient metaClient, HoodieWriteConfig config
    */
   public List<HoodieRollbackStat> performRollback(JavaSparkContext jsc, HoodieInstant instantToRollback, List<RollbackRequest> rollbackRequests) {
 
+    String basefileExtension = metaClient.getTableConfig().getBaseFileFormat().getFileExtension();
     SerializablePathFilter filter = (path) -> {
-      if (path.toString().contains(".parquet")) {
+      if (path.toString().contains(basefileExtension)) {
```

Comment:
Have you tried to fish out all such occurrences in this pr. This is great!

Reply:
Except for tests I have removed all occurrences. Left a few TODOs for test functions which can be removed.

Reply:
Can we also get rid of those in this pr itself

Reply:
@prashantwason : Can you confirm if all occurrences (including tests) are taken care of ?

Reply:
Created Jira to tack this https://issues.apache.org/jira/browse/HUDI-1055